samples/CallWithChat/src/app/views/CallScreen.tsx

```diff
@@ -141,6 +142,18 @@ export const CallScreen = (props: CallScreenProps): JSX.Element => {
           console.log(`Call Id: ${callIdRef.current}`);
         }
       });
+
+      /* send add participant request to chat only after call is connected */
+      const joinThreadWhenCallConnected = async (): Promise<void> => {
+        if (adapter.getState().call?.state === 'Connected' && locator && 'chatThreadId' in locator) {
```

Comment:
Can we join in the "connecting" state? Just as an optimization that it's joined once the user lands in the call